### PR TITLE
Setup ts project

### PR DESCRIPTION
**Pull request by topsart**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.eslintcache
+.env
+.env.test
+node_modules/
+coverage/
+build/
+husky.sh
+yarn.lock
```
Comment (kerrimov) [inline .gitignore:8]: please also include `yarn-debug.log*, yarn-error.log*` to the `gitignore`

File: README.md [hunk 2]
```diff
@@ -1 +1,4 @@
-In progress!
+This is a web application where you can find and view news, events and
```
Comment (kerrimov) [inline README.md:1]: Please add info on how to start the project, run tests

File: package.json [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    "gh-pages": "^3.2.3",
+    "material-ui": "^0.20.2",
+    "react": "^18.0.0",
+    "react-dom": "^18.0.0",
+    "react-router": "^6.3.0",
+    "react-router-dom": "^6.3.0",
+    "react-scripts": "5.0.0",
+    "redux": "^4.1.2",
+    "typescript": "^4.4.2",
+    "web-vitals": "^2.1.0"
```
Comment (kerrimov) [inline package.json:24]: This package should be removed

File: package.json [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+  },
+  "devDependencies": {
+    "eslint": "^8.12.0",
+    "eslint-config-prettier": "^8.5.0",
+    "husky": "^7.0.4",
+    "lint-staged": "^12.3.7",
+    "prettier": "^2.6.2"
+  },
+  "lint-staged": {
+    "*.{ts,css,md}": "prettier --write"
```
Comment (kerrimov) [inline package.json:61]: what about `.tsx` files?

File: public/index.html [hunk 5]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>SSSpace</title>
```
Comment (kerrimov) [inline public/index.html:7]: title should be `The ssspace`

File: src/index.css [hunk 6]
```diff
@@ -0,0 +1,13 @@
+body {
```
Comment (kerrimov) [inline src/index.css:1]: Add `scss` to the project and use it for styles. And also unnecessary styles should be removed.

**Review by kerrimov — CHANGES_REQUESTED** [review]
`favicon.ico` should be removed from the project

File: .gitignore (hunk 1 above)
Comment (topsart) [reply .gitignore:8]: done

File: package.json (hunk 3 above)
Comment (topsart) [reply package.json:24]: Are you talking about web-vitals?

File: package.json (hunk 4 above)
Comment (topsart) [reply package.json:61]: added

File: public/index.html (hunk 5 above)
Comment (topsart) [reply public/index.html:7]: fixed

File: src/index.css (hunk 6 above)
Comment (topsart) [reply src/index.css:1]: done

File: README.md (hunk 2 above)
Comment (kerrimov) [reply README.md:1]: Please beautify readme with formatting

File: README.md [hunk 7]
```diff
@@ -1 +1,17 @@
-In progress!
+## This is a web application where you can find and view news, events and astronauts of different space agencies.
+
+Uses [TheSpaceDevs API](https://thespacedevs.com/llapi)
```
Comment (kerrimov) [inline README.md:3]: 1. Please add `The ssspace` to `readme` as the project name
2. `TheSpaceDevs` => `The Space Devs`
2. add info about `yarn build` command

**Review by kerrimov — APPROVED** [review]

**Pull request merged**